vnext/Microsoft.ReactNative/CompositionSwitcher.idl

```diff
@@ -128,7 +142,9 @@ namespace Microsoft.ReactNative.Composition
       Windows.Graphics.DirectX.DirectXAlphaMode alphaMode);
 
     SpriteVisual CreateSpriteVisual();
+    ShapeVisual CreateShapeVisual();
```

Comment:
If you've got the CreateActivityVisual as part of this interface, then you probably do not need the CreateShapeVisual.  Your implementation of CreateActivityVisual can directly create wincomp shape visuals.

Reply:
good to know! Switched to use SpriteVisual instead